packages/patternfly-3/patternfly-react/src/components/DateTimePicker/DatePicker.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormControl, InputGroup, Icon, OverlayTrigger, Popover } from '../../';
+import DateInput from './DateComponents/DateInput';
+import TodayButton from './DateComponents/TodayButton';
+
+class DatePicker extends React.Component {
+  state = {
+    value: new Date(this.props.value)
+  };
+  formatDate = () => {
```

Comment:
this function has been used in several places over the code, 
could you move it into an `helper` file and extract it from there ?

Reply:
Its different in every place (one for date, one for time, one for date-time) , and exporting it and adding its different variations will make it too complicated in my opinion

Reply:
got it.. thanks